### PR TITLE
SOL-31 [FE] Create markup for update campaign page

### PR DESCRIPTION
**Pull request by jasonchuasun**
## Issue Link
- Backlog [link](https://framgiaph.backlog.com/board/SPH_SOLIDITY?selectedIssueKey=SPH_SOLIDITY-32&issueType=715986)

## Target Page | Component | Function
- `/edit-campaign`

## Pre-condition
- must have the necessary packages installed - `npm install`

## Steps | Commands
1. run the server - `cd fe` & `npm run dev`
2. `visit /edit-campaign` page

## Expected Output
- lazy-validation should be working
- navbar, slot, and footer layouts should display properly 

## Definition of Done
- markup and form validation is working

## Test View Points
- [ ] Add here

## Notes
additional updates include: 
- fixing ESLint and prettier conflicting rules
- navbar improvement
- home page campaign cards spacing

## Screenshots | Recordings
screenshot - image
video - https://github.com/framgia/sph-solidity/assets/118784390/3b6b4834-c0f9-4429-95df-5d1e186034ef

### General
- [ ] Does the code meet the requirements of the project?
- [ ] Is the code readable and maintainable?
- [ ] Are there any unnecessary or redundant lines of code?
- [ ] Are there any long methods or functions that should be refactored?
- [ ] Are there any performance issues?

### Functionality
- [ ] Are there any logical errors?
- [ ] Are edge cases handled properly?
- [ ] Are there any potential security vulnerabilities?

### Code Style
- [ ] Does the code adhere to the project's coding standards?
- [ ] Are variable and function names descriptive and meaningful?
- [ ] Is there proper use of comments and documentation?
- [ ] Is the code properly formatted? Are indentation, spacing, and formatting consistent?

### Security
- [ ] Is the website secure from common attacks?
- [ ] Are inputs properly validated and sanitized?
- [ ] Are sensitive data and credentials properly protected?

### Testing
- [ ] Are there sufficient test cases?
- [ ] Do the test cases cover all functionality?
- [ ] Are the test cases properly written and documented?

### Additional Comments
- [ ] Are there any other issues or concerns with the code?
- [ ] Are there any suggestions for improvement?
- [ ] Are there any positive aspects of the code that should be highlighted?

### CODE REVIEW

**Review by kenorenzsun — DISMISSED** [review]

**Review by johnpaul-sun — DISMISSED** [review]
Approved ✅

Review Time: 15 mins
Comments: 1
IT: 0

**Review by kenorenzsun — DISMISSED** [review]

**Comment by ipilikent** [comment]
## Approved PR
-  All displayed the same as the Figma design
-  The validation messages are displayed
-  Can input the fields
-  Can select date
-  Display up & down arrow in Goal
-  Display back icon
-  Display image
-  Display edit & update campaign button
-  Display footer
-  Navbar improvement
-  Campaign cards spacing improvement

## Proof
<details>

Sol- 31 - FE Update campaign

image

</details>

Test Case: https://docs.google.com/spreadsheets/d/114GrEn-LQQ8saIvjz9zqenuk05qmO4dN/edit#gid=896446223
-----
- Total Test time: 1 hr.
- Total Comments: 0
- Total Iterations: 0

**Review by ipilikent — APPROVED** [review]

**Review by ipilikent — APPROVED** [review]

File: fe/src/components/Navbar.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <header class="shadow-sm bg-white h-16 py-3 px-16">
+  <header class="shadow-sm bg-white/30 h-16 py-3 px-16">
```
Comment (chevyB) [inline fe/src/components/Navbar.vue:2]: is this correct `/30`?

File: fe/src/pages/edit-campaign.vue [hunk 2]
```diff
@@ -0,0 +1,130 @@
+<template>
+  <div class="flex-grow bg-linear-gradient-white-to-light">
+    <div class="max-w-[960px] mx-auto space-y-6 py-6">
+      <div class="flex items-center space-x-2">
+        <BaseButton @click="handleBack">
+          <Icon name="material-symbols:chevron-left" class="h-6 w-6" />
+        </BaseButton>
+        <div class="font-bold text-2xl text-dark">EDIT CAMPAIGN</div>
+      </div>
+      <div class="relative h-60 z-0">
```
Comment (chevyB) [inline fe/src/pages/edit-campaign.vue]: is `z-0` really needed?

File: fe/src/components/Navbar.vue (hunk 1 above)
Comment (jasonchuasun) [reply fe/src/components/Navbar.vue:2]: yes sir. that's for bg-opacity lang.

File: fe/src/pages/edit-campaign.vue (hunk 2 above)
Comment (jasonchuasun) [reply fe/src/pages/edit-campaign.vue]: error on my end. will resolve.

**Review by chevyB — APPROVED** [review]

**Review by johnpaul-sun — APPROVED** [review]
Approved ✅

Review Time: 35 mins
Comments: 0
IT: 0

**Pull request merged**